package com.revature.exceptions;

/**
 * Custom exceptions for handling Location objects. These exceptions are
 * intended to be thrown when attempting to retrieve a Location record from
 * the AssignForce database that does not exist.
 */
public class LocationNotFoundException extends Exception{

}
